pkg/types/helm/provenance_test.go

```diff
@@ -37,12 +38,16 @@ func TestProvenance(t *testing.T) {
 		t.Fatalf("unmarshal error: %v", err)
 	}
 
-	checksum, err := provenance.GetChartHash()
+	algorithm, checksum, err := provenance.GetChartAlgorithmHash()
 
 	if err != nil {
 		t.Fatalf("Error retrieving chart hash: %v", err)
 	}
 
+	if models.HelmV001SchemaChartHashAlgorithmSha256 != algorithm {
+		t.Fatalf("Unexpected checksum algorithm. Expected %s, found %s", models.AlpineV001SchemaPackageHashAlgorithmSha256, algorithm)
```

Comment:
```suggestion
		t.Fatalf("Unexpected checksum algorithm. Expected %s, found %s", models. HelmV001SchemaChartHashAlgorithmSha256, algorithm)
```

Reply:
sigh. updated. good old copy/paste similar to the the prior issue